stargz/stargz_test.go

```diff
@@ -265,6 +268,15 @@ func countGzStreams(t *testing.T, b []byte) (numStreams int) {
 	}
 }
 
+func digestFor(t *testing.T, content string) string {
+	h := sha256.New()
+	_, err := h.Write([]byte(content))
+	if err != nil {
+		t.Fatalf("digestFor: %v", err)
+	}
+	return fmt.Sprintf("sha256:%x", h.Sum(nil))
```

Comment:
Just use https://golang.org/pkg/crypto/sha256/#Sum256 and then you can drop the previous 5 lines and not require a `*testing.T` be passed in